src/networkprovider/ovs_test.go

```diff
@@ -99,9 +84,20 @@ func (suite *NetworkTestSuite) SetupSuite() {
 		NodeName: nodeName,
 	}
 
-	np, err := GetNetworkProvider(&suite.network)
-	suite.NoError(err)
-	suite.np = np.(OVSNetworkProvider)
+	suite.clusterNetwork = entity.Network{
+		Name: tName,
+		OVS: entity.OVSNetwork{
+			BridgeName:    tName,
+			PhysicalPorts: []entity.PhysicalPort{},
+		},
+		Type:        "ovs",
```

Comment:
We'd better use type definition for `Network.Type` in case of typos and invalid values.

```go
type NetworkType string
var OVSNetwork = NetworkType("ovs")
...
```

What do you think?